kolibri/core/kolibri_plugin.py

```diff
@@ -96,7 +96,17 @@ def render_to_page_load_sync_html(self):
 
     @property
     def plugin_data(self):
+        language_code = get_language()
+        static_root = static("assets/fonts/noto-full")
+        full_file = "{}.{}.{}.css?v={}"
         return {
+            "fullCSSFileModern": full_file.format(
+                static_root, language_code, "modern", kolibri.__version__
+            ),
+            "fullCSSFileBasic": full_file.format(
+                static_root, language_code, "basic", kolibri.__version__
+            ),
+            "unsupportedUrl": reverse("kolibri:core:unsupported"),
```

Comment:
This needs to be associated with the bundle that it was originally associated with - this variable is used by the browser minimum requirements check.